apps/services/auth-public-api/src/app/modules/delegation-scopes/delegation-scope.module.ts

```diff
@@ -0,0 +1,14 @@
+import { Module } from '@nestjs/common'
+import { SequelizeModule } from '@nestjs/sequelize'
+import {
+  DelegationScopeService,
+  DelegationScope,
+} from '@island.is/auth-api-lib'
+import { DelegationScopeController } from './delegation-scope.controller'
+
+@Module({
+  imports: [SequelizeModule.forFeature([DelegationScope])],
+  controllers: [DelegationScopeController],
+  providers: [DelegationScopeService],
+})
+export class DelegationScopeModule {}
```

Comment:
Are you adding "scope metadata" endpoints elsewhere? I.e. to query the list of scopes, with translations, that the user can pick when creating custom delegations?